input/fsh/CodeSystem-Mondo.fsh

```diff
@@ -0,0 +1,13 @@
+CodeSystem: Mondo
```

Comment:
Could you change this to `MONDO` as well as the file name?

Reply:
Personally, I think it's preferable to use the correct short form as described on the official website. I'm agreeable to change if you want, but it seems better to use it as it is.

Reply:
I though `MONDO` was official! Ignore this :)